backend/src/core/middlewares/api-key.middleware.ts

```diff
@@ -0,0 +1,50 @@
+import { loggerWithLabel } from '@core/logger'
+import { Handler, NextFunction, Request, Response } from 'express'
+import { ApiKeyService, AuthService } from '@core/services'
+
+export interface ApiKeyMiddleware {
+  deleteApiKey: Handler
+}
+export const InitApiKeyMiddleware = (authService: AuthService) => {
+  const logger = loggerWithLabel(module)
+
+  const deleteApiKey = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+  ): Promise<Response | void> => {
+    try {
+      // enforce authentication by cookie
+      const authenticated = authService.checkCookie(req)
+      if (!authenticated) {
+        return res.sendStatus(403).json({
+          message: `Invalid cookie`,
```

Comment:
add field `code: unauthorized` as we're getting to [this](https://www.notion.so/API-Responses-in-JSON-5174d66bf5ce49a1b3d161630869eb65) soon

Reply:
this part has been removed, using authMiddleware now